src/olympia/scanners/models.py

```diff
@@ -94,6 +94,24 @@ def get_scanner_name(self):
     def get_pretty_results(self):
         return json.dumps(self.results, indent=2)
 
+    def get_files_by_matched_rules(self):
+        res = dict()
```

Comment:
you can simplify below by using `defaultdict(list)`

Reply:
oh, I did not know that, thanks!